commands/channel_test.go

```diff
@@ -2176,3 +2142,165 @@ func (s *MmctlUnitTestSuite) TestRenameChannelCmd() {
 		s.Require().Equal(printer.GetLines()[0], updatedChannel)
 	})
 }
+
+func (s *MmctlUnitTestSuite) TestRemoveChannelUsersCmd() {
+	mockUser := model.User{Id: userID, Email: userEmail}
+	argsTeamChannel := teamName + ":" + channelName
+
+	s.Run("should remove user from channel", func() {
+		printer.Clean()
+
+		cmd := &cobra.Command{}
+		args := []string{argsTeamChannel, userEmail}
+
+		foundTeam := &model.Team{
+			Id:          teamID,
+			DisplayName: teamDisplayName,
+			Name:        teamName,
+		}
+
+		foundChannel := &model.Channel{
+			Id:          channelID,
+			Name:        channelName,
+			DisplayName: channelDisplayName,
+		}
+
+		s.client.
+			EXPECT().
+			GetTeam(teamName, "").
+			Return(foundTeam, &model.Response{Error: nil}).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetChannelByNameIncludeDeleted(channelName, foundTeam.Id, "").
+			Return(foundChannel, &model.Response{Error: nil}).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetUserByEmail(userEmail, "").
+			Return(&mockUser, &model.Response{Error: nil}).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			RemoveUserFromChannel(foundChannel.Id, mockUser.Id).
+			Return(true, &model.Response{Error: nil}).
+			Times(1)
+
+		err := removeChannelUsersCmdF(s.client, cmd, args)
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetLines(), 0)
+	})
+
+	s.Run("should throw error if both --all-users flag and user email are passed", func() {
+		printer.Clean()
+
+		cmd := &cobra.Command{}
+		cmd.Flags().Bool("all-users", true, "Remove all users from the indicated channel.")
+		args := []string{argsTeamChannel, userEmail}
+
+		err := removeChannelUsersCmdF(s.client, cmd, args)
+		s.Require().EqualError(err, fmt.Sprintf("individual users must not be specified in conjunction with the --all-users flag"))
+	})
+
+	s.Run("should remove all users from channel", func() {
+		printer.Clean()
+
+		cmd := &cobra.Command{}
+		cmd.Flags().Bool("all-users", true, "Remove all users from the indicated channel.")
+		args := []string{argsTeamChannel}
+
+		foundTeam := &model.Team{
+			Id:          teamID,
+			DisplayName: teamDisplayName,
+			Name:        teamName,
+		}
+
+		foundChannel := &model.Channel{
+			Id:          channelID,
+			Name:        channelName,
+			DisplayName: channelDisplayName,
+		}
+
+		mockMember := model.ChannelMember{ChannelId: channelID, UserId: mockUser.Id}
+		mockChannelMembers := model.ChannelMembers{mockMember, mockMember, mockMember}
```

Comment:
Can we change this members list to have three different members? I think the current one could eventually mask errors in the test